src/routes/dev/columns.svelte

```diff
@@ -1,127 +0,0 @@
-<script>
```

Comment:
Is there a requirement to delete this file?

Reply:
That file imported the `NavEntry` component, so after this PR's first commit, the file wasn't compiling anymore and it broke the CI/CD builds. Rather than refactor the prototype (from 4 months ago), I elected to just delete it.

Reply:
In that case probably @hamilton can let us know if we still need this. IIRC he mentioned that he was using bits and pieces from dev for integrations, testing new interactions and prototyping new stuff

Reply:
Just got a confirmation from Hamilton that we're good to delete this 👍 . We'll have to start formalizing this process, but any `dev` route that is considerably old/outdated is fine to delete